scene/3d/collision_object_3d.cpp

```diff
@@ -62,6 +77,8 @@ void CollisionObject3D::_notification(int p_what) {
 				PhysicsServer3D::get_singleton()->body_set_state(rid, PhysicsServer3D::BODY_STATE_TRANSFORM, get_global_transform());
 			}
 
+			_on_transform_changed();
```

Comment:
Why is this call needed here? It seems `NOTIFICATION_ENTER_TREE` should already initialize debug shapes with the right transform, but I might be missing a specific case to handle.

Reply:
Debug shapes don't auto update when the CollisionObject transform changes (and you ask me to do that in [this comment](https://github.com/godotengine/godot/pull/48175#pullrequestreview-646202859))

Reply:
Oh you're right, sorry. I confused myself because a part of the code was clipped and I thought it was in `NOTIFICATION_ENTER_WORLD` :)